assets/xml/objects/object_ob.xml

```diff
@@ -1,84 +1,100 @@
+        <Texture Name="gMoonChildTwinmoldsMaskNoseTex" OutName="moonchild_twinmolds_mask_nose" Format="ci4" Width="64" Height="64" Offset="0xFB40" />
+
+        <!-- Moonchild Moving Animations -->
+        <Animation Name="gMoonChildWalkingAnim" Offset="0x10880" />
+        <Animation Name="gMoonChildRunningAnim" Offset="0x10CD8" />
+
+        <!-- Moonchild DisplayLists -->
+        <DList Name="gMoonChildPelvisDL" Offset="0x14190" />
+        <DList Name="gMoonChildLeftThighDL" Offset="0x142C0" />
+        <DList Name="gMoonChildLeftLegDL" Offset="0x144D0" />
+        <DList Name="gMoonChildLeftFootDL" Offset="0x14638" />
+        <DList Name="gMoonChildRightThighDL" Offset="0x147A0" />
+        <DList Name="gMoonChildRightLegDL" Offset="0x149B0" />
+        <DList Name="gMoonChildRightFootDL" Offset="0x14B18" />
+        <DList Name="gMoonChildChestBottomDL" Offset="0x14C80" />
+        <DList Name="gMoonChildChestDL" Offset="0x14DA0" />
+        <DList Name="gMoonChildHeadDL" Offset="0x14F18" />
+        <DList Name="gMoonChildLeftArm" Offset="0x15380" />
+        <DList Name="gMoonChildLeftForeArm" Offset="0x154B0" />
+        <DList Name="gMoonChildLeftHand" Offset="0x155D0" />
+        <DList Name="gMoonChildRightArm" Offset="0x15758" />
+        <DList Name="gMoonChildRightForeArm" Offset="0x15888" />
+        <DList Name="gMoonChildRightHand" Offset="0x159A8" />
+
+        <!-- Moonchild Textures -->
+        <Texture Name="gMoonChildTLUT" OutName="moonchild_tlut" Format="rgba16" Width="16" Height="16" Offset="0x15B28" />
+        <Texture Name="gMoonChildFingersTex" OutName="moonchild_fingers" Format="ci8" Width="16" Height="16" Offset="0x15D28" />
+        <Texture Name="gMoonChildThighSkinTex" OutName="moonchild_thigh_skin" Format="ci8" Width="8" Height="8" Offset="0x15E28" />
+        <Texture Name="gMoonChildTshirtDetailTex" OutName="moonchild_tshirt_detail_tex" Format="ci8" Width="16" Height="32" Offset="0x15E68" />
+        <Texture Name="gMoonChildHairTex" OutName="moonchild_hair" Format="ci8" Width="8" Height="16" Offset="0x16068" />
+        <Texture Name="gMoonChildEarTex" OutName="moonchild_ear" Format="ci8" Width="8" Height="8" Offset="0x160E8" />
+        <Texture Name="gMoonChildBackHair" OutName="moonchild_back_hair" Format="ci8" Width="8" Height="16" Offset="0x16128" />
+        <Texture Name="gMoonChildTshirtNecklineTex" OutName="moonchild_tshirt_neckline" Format="ci8" Width="16" Height="32" Offset="0x161A8" />
+
+        <!-- Moonchild Limbs -->
+        <Limb Name="gMoonChildRootLimb" Type="Standard" EnumName="MOONCHILD_LIMB_ROOT" Offset="0x163A8" />
+        <Limb Name="gMoonChildBottomChestLimb" Type="Standard" EnumName="MOONCHILD_LIMB_BOTTOM_CHEST" Offset="0x163B4" />
+        <Limb Name="gMoonChildUpperChestLimb" Type="Standard" EnumName="MOONCHILD_LIMB_UPPER_CHEST" Offset="0x163C0" />
+        <Limb Name="gMoonChildLeftArmLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_ARM" Offset="0x163CC" />
```

Comment:
While "arm" is the correct medical term for this part, we tend to use "upper arm" here, since oftentimes "arm" is used to refer to the whole arm + forearm + hand
```suggestion
        <Limb Name="gMoonChildLeftUpperArmLimb" Type="Standard" EnumName="MOONCHILD_LIMB_LEFT_UPPER_ARM" Offset="0x163CC" />
```